src/components/LastCheckedDisplay.tsx

```diff
@@ -11,7 +11,7 @@ export interface LastCheckedDisplayProps {
 
 export const LastCheckedDisplay = ({textDark}: LastCheckedDisplayProps) => {
   const i18n = useI18n();
-  const [exposureStatus] = useExposureStatus();
+  const exposureStatus = useExposureStatus();
```

Comment:
This file is no longer in use.

Reply:
I'll remove it in a separate PR.

Reply:
Added a PR here - https://github.com/cds-snc/covid-alert-app/pull/1218